spec/requests/v6/assessments_controller_spec.rb

```diff
@@ -437,6 +447,50 @@ def cash_transactions(amount)
         end
       end
 
+      context "with employment income without payments" do
+        let(:employed) { true }
+        let(:params) { { employment_income: employment_income_without_payments_params } }
+
+        it "returns http success" do
+          expect(response).to have_http_status(:success)
+        end
+
+        describe "disposable_income from summary" do
+          let(:employment_income) { parsed_response.dig(:result_summary, :disposable_income, :employment_income) }
+
+          it "has employment income" do
```

Comment:
I think it would be good if this test highlighted (and maybe only tested for) the key point - which is that the employment deduction is zero in this case